.travis.yml

```diff
@@ -6,8 +6,7 @@ otp_release:
 addons:
   - postgresql: '9.5'
 env:
-  - MIX_ENV=test
-  - PG_USER=postgres
+  - MIX_ENV=test PG_USER=postgres
```

Comment:
Also, this line might have been better suited in a separate PR, but this changes the travis job from running two steps and combines the config it into a single job.  When you review details, since there is only one thing to do it shows the output automatically.  Kind of a nice win.